tools/check_repositories.sh

```diff
@@ -12,7 +12,7 @@ fi
 
 # Check whether number of defined `url =` or `urls =` and `sha256 =` kwargs in
 # repository definitions is equal.
-urls_count=$(grep -IPnrs "url(s)? =" --include=*.bzl . | wc -l)
+urls_count=$(egrep -Inrs "url(s)? =" --include=*.bzl . | wc -l)
```

Comment:
as you are already here, can you change this to
`git grep -E "url(s)? =" -- '*.bzl' | wc -l`
and the line below to `git grep` as well?

git grep is consistent across platform, and won't traverse into the files in bazel generated symoblic link directories.

Reply:
Hmm, seems like this is causing some issues
```bash
Found more defined repository URLs than SHA256 sums, which means that there are some repositories without sums.
```

Reply:
Looking at the commit, https://github.com/envoyproxy/envoy/commit/02a76e37abd83c1530a8c1a1f28d980789deff85#diff-46076f84962fa9bdebb811d599114191R16

Remove . at the end of git grep? It will include this shell script otherwise.

Reply:
ah good catch! I completely missed that